### DISCUSSION TITLE
Removing author information from distill style post

### DISCUSSION THREAD

**Discussion by prateekgargX** [alshedivat/al-folio#1083] [Q&A]
Is there a way to complete the author's information from the header, since on a personal blog, you'll self-publish most of the times

**Comment by riasat-sheikh** [comment]
#### Permanently add your name
To do this I would suggest you to modify the author section inside the `<d-front-matter>` of `_layouts/distill.html` file as below

```json
"authors": [
  {
    "author": "{{ site.first_name }} {{site.middle_name}} {{ site.first_name }}", // arrange this accordingly whatever you need
    "authorURL": "{{ site.url }}",
    "affiliations": [
      {
        "name": "{{ site.data.distill.affiliations.name }}",
        "url": "{{ site.data.distill.affiliations.url }}"
      }
    ]
  }
],
```

Also, make a new file called `distill.yml` inside the `_data` folder and add the following,

```yaml
affiliations:
  name: "your current affiliation name"
  url: "the affiliation url if any"
```

This will take care of the name and url in the distill.html file and you won't have to come back here to update every time. You can just edit inside the data folder.

#### Completely remove that section

To completely remove that section you just need to comment out the `<d-byline></d-byline>` under the content section. It should look something like this

```html
    <!-- Content -->
    <div class="post distill">

      <d-title>
        <h1>{{ page.title }}</h1>
        <p>{{ page.description }}</p>
      </d-title>

      <!-- <d-byline></d-byline> -->

      <d-article>
```

**Comment by QingXin96** [comment]
how to center the title in the distill style blog, i try to do like this,


    <div class="post distill">

      <d-title>
        <h1><center>{{ page.title }}</center></h1>
        <p><center>{{ page.description }}</center></p>
      </d-title>

      

      <d-article>

The title is fine, while the description not fit! Anyone knows how to do it? I am completely a beginner!

**Reply by riasat-sheikh** [reply]
A screenshot about the `description not fit!` would have been better but may you try changing to the below code.

```html
<h1 class="text-center">{{ page.title }}</h1>
<p class="text-center">{{ page.description }}</p>
```

I hope this is what you want.

<img width="1208" alt="image" src="https://github.com/alshedivat/al-folio/assets/63984668/3ef2233f-6152-4757-a276-3e1b4c2b6ad7">
